### ISSUE TITLE
Refactor fluid fonts

### ISSUE THREAD

**Issue by manolobevia** [cul-it/signage#31]
The current approach of using a sass mixin is heavily time consuming for calculating the size of the font, margin, padding, etc of each markup element. Also on mobile devices, the layout breaks because of the size of the fonts. Fonts scales well between 1440px and 1920px width devices, but not in mobile devices. Instead of making fonts fluid for any width device, using this  **[approach](http://stackoverflow.com/questions/5358183/is-it-possible-to-dynamically-scale-text-size-based-on-browser-width)** won't require any font size calculations.

**Comment by manolobevia** [comment]
Tried my own approach:

For making fluid content this is what I have done:

I have designed and done all my calculations in term of font-size, paddings, margins, etc in a viewport size of 1920x1080 for Olin signage. I have also used VW and VH units. 

Since I am working with a 1920px width viewport, 1920px is 100 vw.

1920 px ------> 100vw
       n px ------>     x vw

n = any width (any horizontal size) , ex: 14px for a font-size,
x = the value of n but in vw units

x = n px * 100vw/1920px, we can simplify this into:
x = n * 0.053 vw

so 20 (n) pixel padding-left is 1.06 (x) nw

font-size: calc(20 * 0.053vw); 

For heights like padding-top, margin-top, etc, any vertical size

1080 px ------> 100vh
       n px ------>     y vh

n = any height, ex: 14px for a padding-top,
y = the value of n but in vh units

y= n px * 100vh/1920px, we can simplify this into:
y = n * 0.053 vh
